lib/qif2json.js

```diff
@@ -59,7 +81,10 @@ exports.parse = function(qif, options) {
         }
         switch (line[0]) {
             case 'D':
-                transaction.date = parseDate(line.substring(1), options.dateFormat);
+                transaction.date = parseDate(line.substring(1), options.usDates, options.oldDates);
+                break;
+            case 'U':
+                // Looks like a legacy repeat of 'T'
```

Comment:
No tests for this?

Reply:
Also, i there any docs for this? I can't see in https://www.w3.org/2000/10/swap/pim/qif-doc/QIF-doc.htm

Reply:
The U? Nope, I based this on wikipedia's list of fields. I can add a test